src/custom/constants/index.ts

```diff
@@ -51,9 +51,9 @@ export const GP_VAULT_RELAYER: Partial<Record<number, string>> = {
 }
 
 export const V_COW_CONTRACT_ADDRESS: Partial<Record<number, string>> = {
-  [ChainId.MAINNET]: '0x6d04B3ad33594978D0D4B01CdB7c3bA4a90a7DFe',
-  [ChainId.XDAI]: '0xA3A674a40709A837A5E742C2866eda7d3b35a7c0',
-  [ChainId.RINKEBY]: '0xD7Dd9397Fb942565959c77f8e112ec5aa7D8C92c',
+  [ChainId.MAINNET]: '0x53877cd4111b4c041abea77cc501ed68c22630bd ',
+  [ChainId.XDAI]: '0x7a005ebd31f7c6ee13bf44b936952c3b308d53e0 ',
+  [ChainId.RINKEBY]: '0xa9ad5792a458a212b9e2f004e5663ef77aba6c11',
 }
```

Comment:
I guess you need to use checksummed addresses, tests are failling

Reply:
Doesn't look that was the issue, still failing :/

Reply:
no, I think it was. Let it finish now :) 
i've made a mistake on the push. too late for me